corehq/apps/app_manager/templates/app_manager/v2/partials/case_list.html

```diff
@@ -76,6 +79,34 @@ <h4 class="panel-title panel-title-nolink">
                 </label>
             </div>
         </div>
+        {% endif %}
+
+        {% if request|feature_preview_enabled:"SHARE_CASE_TILE_CONTEXT" %}
+        {{ request|toggle_tag_info:"CASE_LIST_TILE" }}
+        {% if request|toggle_enabled:'CASE_LIST_TILE' %}
+        {% if request|toggle_enabled:'SHOW_PERSIST_CASE_CONTEXT_SETTING' %}
```

Comment:
having something under 2 toggles seems pretty weird. Can we not clean up the toggles so that there's only 1?